checkstyle/checkstyle.xml

```diff
@@ -132,7 +132,7 @@
     </module>
     <module name="NPathComplexity">
       <!-- default is 200 -->
-      <property name="max" value="500"/>
+      <property name="max" value="550"/>
```

Comment:
Why change the setting instead of modifying `suppressions.xml` to exclude certain classes from this rule?